src/com/cft/contactmerge/FirstAndLastNamesMatchLogic.java

```diff
@@ -0,0 +1,40 @@
+package com.cft.contactmerge;
+
+public abstract class FirstAndLastNamesMatchLogic {
+    private final static String[] namePrefixes = {"Mr", "Mrs", "Miss", "Dr"};
```

Comment:
Wouldn't it be better to use the List collection? In C# they encourage us to use collections whenever possible. Not sure if it is the same in Java.